app/handlers/newflow/send_reset_password_email.rb

```diff
@@ -14,11 +14,17 @@ class SendResetPasswordEmail
     protected #################
 
     def authorized?
-      forgot_password_form_params.email.present? || logged_in_user
+      true
     end
 
     def handle
       outputs.email = forgot_password_form_params.email
+
```

Comment:
does handle have a spec to make sure it gets a fatal_error for these conditions?

Reply:
Now it does :) Pushed a commit. Thanks for the suggestion!